ldes-server-integration-test/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/LdesServerSteps.java

```diff
@@ -125,4 +142,16 @@ public void theResponseFromRequestingTheUrlHasAccessControlHeadersAndAnEtag(Stri
 		assertEquals("*", response.getHeader("Access-Control-Allow-Origin"));
 		assertNotNull(response.getHeader("ETag"));
 	}
+
+	@Then("the collection {string} contains {long} members")
+	public void theCollectionContainsMembers(String collection, long expectedMemberCount) {
+		long memberCountForCollection = memberRepository.getMemberStreamOfCollection(collection).count();
+
+		assertEquals(expectedMemberCount, memberCountForCollection);
+	}
```

Comment:
Since, these tests act on the boundaries of the ldes server, it would be better to fetch the page and verify it only contains 10 members instead of checking the database which is something internal that an external customer would not have access to.

Reply:
Was my first thought too.. yet the problem is.. Since we plan on making page Ids UUIDs, getting such page ids will be tricky..

Reply:
Although I follow your reasoning, I would make an exception for this. 
Retrieving the members of a collection would only be possible by first calling the view page, then follow its relations until we get members, ... only to then get the count. 
I think following the LDES itself is something the Client does, so I would take this shortcut for the integration tests on this level